src/app/dashboard/whitelist/actions.js

```diff
@@ -0,0 +1,273 @@
+//@flow
+
+import * as ui from 'polymath-ui'
+import { TransferManager, SecurityToken } from 'polymathjs'
+import type { Investor } from 'polymathjs/types'
+import { formName as userFormName } from './components/userForm'
+import { formName as editInvestorsFormName } from './components/editInvestorsForm'
+import type { GetState } from '../../../redux/reducer'
+import type { ExtractReturn } from '../../../redux/helpers'
+
+//ac_ = actionCreator
+export const TRANSFER_MANAGER = 'dashboard/whitelist/TRANSFER_MANAGER'
+export const ac_transferManager = (transferManager: TransferManager) => ({ type: TRANSFER_MANAGER, transferManager: transferManager })
+
+export const UPLOAD_CSV = 'dashboard/whitelist/UPLOAD_CSV'
+export const ac_csvUpload = (csvMessage: string, addresses: Array<string>, sell: Array<string>, buy: Array<string>, previewCSVShowing: boolean, ) => ({ type: UPLOAD_CSV, csvMessage, addresses, sell, buy, previewCSVShowing })
```

Comment:
String is too long, longer than 120 columns. This also should have been warned by our eslint-plugin

Reply:
fixed.

made a pull request to add this rule to eslint https://github.com/PolymathNetwork/eslint-plugin-polymath-react/pull/1